resources/views/livewire/order/additional-addresses.blade.php

```diff
@@ -0,0 +1,60 @@
+<div class="flex flex-col gap-4">
+    <div wire:ignore>
+        @teleport('body')
+            <x-modal name="edit-address-assignment">
+                <x-card>
+                    <div class="flex flex-col gap-4">
+                        <x-select
+                            :label="__('Address')"
+                            option-value="id"
+                            option-label="label"
+                            template="user-option"
+                            wire:model="address_id"
+                            :async-data="[
+                                'api' => route('search', \FluxErp\Models\Address::class),
```

Comment:
can this be any address? Not constraint by contact_id or client_id?

Reply:
can be any address, yes